### DISCUSSION TITLE
Where do I find the settings files of Dark Reader on Chrome?

### DISCUSSION THREAD

**Discussion by SixelAlexiS90** [darkreader/darkreader#8639] [Q&A]
Hi! I want to make a backup of the Dark Reader settings (like all the different profiles for every site I use) due to a PC format I'm going to do in the next days.

I use Dark Reader on Chrome and I know that it's supposed to sync extensions, but the sync of extensions settings need to be supported by the extension itself.
I'm not sure if Dark Reader support sync on that, but last time I've formatted the PC, Dark Reader settings didn't synced up so I want to make a backup just to be sure.

I took a look at:
\AppData\Local\Google\Chrome\User Data\Default\Extensions\eimadpbcbfnmbkopoojfekhnkhdbieeh\4.9.47_0

But I'm not sure if any of those files stores the user preferences.
Thank you for any help :)

**Comment by SixelAlexiS90** [comment]
Since no one has responded, and after the format I got synced with 3 years old Dark Reader settings via Chrome, this is where you find the Dart Reader settings on Chrome if you want to manually backup it and restore:

C:\Users\username\AppData\Local\Google\Chrome\User Data\Default\Local Extension Settings\eimadpbcbfnmbkopoojfekhnkhdbieeh

It's the "eimadpbcbfnmbkopoojfekhnkhdbieeh" folder that contain all the Dark Reader settings.
I don't know if it's my Chrome sync that is broken or it's Dark Reader fault but here it is, hopefully will be helpful for others as well.

**Reply by Gusted** [reply]
> I don't know if it's my Chrome sync that is broken or it's Dark Reader fault but here it is, hopefully will be helpful for others as well.

Your browser is responsible for the saving of user setting to storage and remote servers.

**Reply by SixelAlexiS90** [reply]
> > I don't know if it's my Chrome sync that is broken or it's Dark Reader fault but here it is, hopefully will be helpful for others as well.
> 
> Your browser is responsible for the saving of user setting to storage and remote servers.

Yeah, I suspected in these years that part of my Chrome sync was busted, the annoying part is that I can't cancel only some part of the sync but I have to delete it all. Before next PC format I have to do it hoping extensions like Dark Reader will sync properly. Thank you :)

**Comment by SixelAlexiS90** [comment]
Almost three months later I discover that import/export setting is a thing, and apparently is buried in the new design that is in the dev tools tab.
I had to google and find that in another github topic to discover something I always wanted and was there by two years.
I really dislike the new design but gladly I can go back and forth to the old design.

p.s. It could have been useful to know that some months ago as well instead of messing with files. Not so many "answers" for a Q&A section :'D

**Comment by oeloo** [comment]
@SixelAlexiS90  Thank you, for your useful post. I have discovered this export feature in the new UI interface thanks to this.
Sad that you did not get an answer on your initial question, but maybe this export feature was not yet implemented as it is in the new UI not yet officially used.

**Reply by SixelAlexiS90** [reply]
> @SixelAlexiS90 Thank you, for your useful post. I have discovered this export feature in the new UI interface thanks to this. Sad that you did not get an answer on your initial question, but maybe this export feature was not yet implemented as it is in the new UI not yet officially used.

Hi! Sadly the import/export feature is broken, it implemented me old and partial filters after the import on the new PC... gladly I had saved the Dark Reader folder in "Local Extension Settings", otherwise I would have lost most of my settings.

**Reply by oeloo** [reply]
Thanks I will then do it the safe way, using the local extension settings folder too
